### PR TITLE
Add Default Windows Release Script for Nano

### PR DESCRIPTION
**Pull request by charlieJ107**
Add two defautl windows release script to add the windows release job for nano.

### CODE REVIEW

**Comment by yangw1234** [comment]
let's not change python/dev/release.sh and python/setup.py

**Comment by charlieJ107** [comment]
> let's not change python/dev/release.sh and python/setup.py

Changes has been revert in https://github.com/intel-analytics/BigDL/pull/3584/commits/5aa22152bdf4c53328f4f0a328073165f50210b7

File: python/dev/release.sh [hunk 1]
```diff
@@ -46,6 +46,8 @@ if [ "$platform" ==  "mac" ]; then
     verbose_pname="macosx_10_11_x86_64"
 elif [ "$platform" == "linux" ]; then
     verbose_pname="manylinux1_x86_64"
+elif [ "$platform" == "windows" ]; then
```
Comment (yangw1234) [inline python/dev/release.sh:49]: let's reset the changes in this file.

**Comment by yangw1234** [comment]
http://10.112.231.51:18888/view/ZOO-PR/job/ZOO-PR-NanoTests/99/ jenkins passed

**Pull request merged**